po/pt_BR.po

```diff
@@ -220,7 +220,7 @@ msgstr "Exportar o tema do terminal"
 
 #: oomox_gui/main.py:592
 msgid "Export _Icons"
-msgstr "Exportar _Ícones"
```

Comment:
i was scratching my display for few seconds to remove the dust from `Í` letter :D